src/messages/MMonMap.h

```diff
@@ -48,11 +50,13 @@ class MMonMap final : public Message {
 
     using ceph::encode;
     encode(monmapbl, payload);
+    encode(quorum, payload);
   }
   void decode_payload() override { 
     using ceph::decode;
     auto p = payload.cbegin();
     decode(monmapbl, p);
+    decode(quorum, p);
```

Comment:
I don't see any versioning. `Message` has its own machinery you can control (provide versions) via a ctor, IIRC.

Reply:
right, missed that one for the message